### PR TITLE
feat(middleware): adding sentry as errortracker

### PR DESCRIPTION
**Pull request by Windfarer**
#### Description (what this PR does / why we need it):

add sentry middleware for error tracking

#### Which issue(s) this PR fixes (resolves / be part of):

1. move https://github.com/go-kratos/sentry into the main repo
2. bump the go-sentry version
3. refactoring attributes naming
4. add tags options

#### Other special notes for the reviewers:

The documents should be add to go-kratos.dev and the origin repo https://github.com/go-kratos/sentry should be set to archived

### CODE REVIEW

**Comment by shenqidebaozi** [comment]
Are you ready to merge?

**Comment by shenqidebaozi** [comment]
@Windfarer I have archived https://github.com/go-kratos/sentry

**Comment by Windfarer** [comment]
@shenqidebaozi ready for review

File: contrib/errortracker/sentry/sentry.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	"strings"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+
+	"github.com/go-kratos/kratos/v2/log"
+	"github.com/go-kratos/kratos/v2/middleware"
+	"github.com/go-kratos/kratos/v2/transport"
+	"github.com/go-kratos/kratos/v2/transport/grpc"
+	http2 "github.com/go-kratos/kratos/v2/transport/http"
```
Comment (shenqidebaozi) [inline contrib/errortracker/sentry/sentry.go:17]: Why do we need alias

**Review by shenqidebaozi — APPROVED** [review]

**Review by Cluas — APPROVED** [review]

**Pull request merged**